scripts/deploy/manual-major-prerelease.mjs

```diff
@@ -0,0 +1,73 @@
+import {getCurrentVersion} from '@coveo/semantic-monorepo-tools';
+import {resolve} from 'path';
+import {execute} from '../exec.mjs';
+import {commitVersionBump, stageAll, tagExists, tagPackages} from '../git.mjs';
+import {
+  getPackageDefinitionFromPackageName,
+  updatePackageVersion,
+} from '../packages.mjs';
+import {prereleaseSuffix} from '../prerelease.mjs';
+
+/**
+ * @typedef {import('../packages.mjs').PackageDefinition} PackageDefinition
+ */
```

Comment:
This is the JSDoc equivalent to `import type { PackageDefinition } from '../package';`.

Reply:
nice touch